airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/connector_runner.py

```diff
@@ -130,3 +130,9 @@ def run(self, cmd, config=None, state=None, catalog=None, **kwargs) -> Iterable[
                 yield AirbyteMessage.parse_raw(line)
             except ValidationError as exc:
                 logging.warning("Unable to parse connector's output %s", exc)
+
+    def has_env_var(self, lookup_env_var: str = "AIRBYTE_ENTRYPOINT"):
+        env_vars = self._image.attrs["Config"]["Env"]
+        env_airbyte_entrypoint = [i for i in env_vars if i.startswith(lookup_env_var)]
+        ret = env_airbyte_entrypoint and env_airbyte_entrypoint[0].split(f"{lookup_env_var}=", 1)[1]
+        return bool(ret)
```

Comment:
I think this should be a comparison 
```
env_vars_list = [env.split('=')[0] for env in env_vars]
return lookup_and_var in env_vars_list
```

rename  `env_airbyte_entrypoint` because the function should be generic.
also IMO should remove the default value for `lookup_env_var` variable, people should understand reading the code without recalling the function to know that in default case is looking for the Entrypoint variable.